.github/workflows/deployment.yml

```diff
@@ -0,0 +1,22 @@
+name: Production Tag Deployment
+env:
+  VERCEL_ORG_ID: ${{ secrets.VERCEL_ORG_ID }}
+  VERCEL_PROJECT_ID: ${{ secrets.VERCEL_PROJECT_ID }}
+on:
+  push:
+    # Pattern matched against refs/tags
+    tags:        
+      - '*' # Push events to every tag not containing /
```

Comment:
Kindly change to `v*` to match semantic versioning pattern

Reply:
done, thanks @jofftiquez